editions/tw5.com/tiddlers/filters/syntax/Named Filter Run Prefix.tid

```diff
@@ -1,10 +1,34 @@
 created: 20201214044413473
-modified: 20211118025845599
-tags: [[Filter Syntax]]
+modified: 20230319182233796
+tags: [[Filter Run Prefix]]
 title: Named Filter Run Prefix
+type: text/vnd.tiddlywiki
 
-<$set name="prefixlist" filter="""[all[shadows+tiddlers]has[module-type]module-type[filterrunprefix]trim:prefix[$:/core/modules/filterrunprefixes/]trim:suffix[.js]addprefix["]addsuffix["]join[|]addprefix[(]addsuffix[)]]""">
-<$railroad text=<<prefixlist>>/>
-</$set>
+Parents: <$macrocall $name=".recursiveParentTag" current=<<currentTiddler>> stop="Filters"/>
 
-A named filter run prefix can precede any [[run|Filter Run]] of a [[filter expression|Filter Expression]] in place of a single-character prefix (`+`, `-` and so on). To create a new filter run prefix, create a [[Javascript module|Modules]] with a [[module-type|ModuleType]] of `filterrunprefix`.
+The first named prefixes for filter runs are available since: <<.from-version "5.1.23">> The exact version can be seen at the detailed description of each prefix.
```

Comment:
That information is not useful here, as I have to look it up elsewhere anyway.

Reply:
Yea, but if not mentioned here, users may just all the named prefixes listed here. But not all of them will work with older TW versions. ... So there must be some info about different implementation timelines somewhere in a prominent place.

Reply:
I did add some comments in the diagrams. I think you'll like it